.github/workflows/github-actions.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       # If you want to matrix build , you can append the following list.
       matrix:
         go_version:
-          - 1.17
+          - 1.20
```

Comment:
There is bug in github when we configure golang version great than or equals to 1.20.
<img width="841" alt="image" src="https://github.com/apache/dubbo-go/assets/33331974/bdce17d7-87eb-47bc-81d6-d879e938193b">
We should add quote on this golang version:
```
'1.20'
```